src/Payments/PreMountLoader.res

```diff
@@ -1,29 +1,45 @@
 let sendPromiseData = (promise, key) => {
-  open Promise
-  promise
-  ->then(res => resolve(res))
-  ->catch(_ => resolve(JSON.Encode.null))
-  ->thenResolve(response => {
-    Utils.messageParentWindow([("response", response), ("data", key->JSON.Encode.string)])
-  })
-  ->ignore
+  (
+    async () => {
+      let response = try {
+        await promise
+      } catch {
+      | _ => JSON.Encode.null
+      }
+      Utils.messageParentWindow([("response", response), ("data", key->JSON.Encode.string)])
+    }
+  )()->ignore
```

Comment:
maybe define a named fn like
```
let something = async () => {
}

something()->ignore
```
instead of defining and calling the fn immediately